Sprint 2/issue 14#create sign in screen

Antonio3veces3 opened the pull request:
# Description
In this branch we create the Sign in Screen, where the user has two options to sign in, their can sign in with Google and Facebook. 
This code is only design, doesn't have functionality.

What did you implemented/Why did you implemented this:

- Create the design of SignIn Screen
- Create buttons to sign in with Google
- Create buttons to sign in with Facebook
- Add icon of Google and Facebook into the buttons.


## Screenshots

In this image we can se the finished Sign In screen.
This is the result:
![signInScreen](https://github.com/BrightCoders-Institute/proyecto-final-oppenbarbies/assets/60717103/374ebc87-9943-471e-93d4-ff1a06c0bd5f)

Antonio3veces3 committed:
Initial commit to start coding

Antonio3veces3 committed:
Create Sign in Screen

Create screen design and create the sign in button as a reusable
component, this means, with a component we create both google
button and facebook button to sign in. Here is only design, it
havent functionality.

Antonio3veces3 committed:
Change  icons button

Change icon button to the original icon of authProvider.
Update de google icon and facebook icon.

Antonio3veces3 committed:
merge conflicts resolved

Antonio3veces3 committed:
SignIn screen finished

Lizzflores reviewed (approved):
LGTM

AlbertoG22 reviewed (approved):
Nice 🚀

BrandonVG commented on CtasApp/yarn.lock:
```diff
@@ -3197,11 +3212,24 @@ eslint-visitor-keys@^2.1.0:
+  resolved "https://registry.npmjs.org/eslint-visitor-keys/-/eslint-visitor-keys-3.4.2.tgz"
+  integrity sha512-8drBzUEyZ2llkpCA67iYrgEssKDUu68V8ChqqOfFupIaG/LCVPUT+CoGJpT77zJprs4T/W7p07LP7zAIMuweVw==
+
+<<<<<<< HEAD
+eslint-visitor-keys@^3.4.1:
+  version "3.4.2"
+  resolved "https://registry.npmjs.org/eslint-visitor-keys/-/eslint-visitor-keys-3.4.2.tgz"
+  integrity sha512-8drBzUEyZ2llkpCA67iYrgEssKDUu68V8ChqqOfFupIaG/LCVPUT+CoGJpT77zJprs4T/W7p07LP7zAIMuweVw==
+
+eslint-visitor-keys@^3.4.2:
   version "3.4.2"
   resolved "https://registry.npmjs.org/eslint-visitor-keys/-/eslint-visitor-keys-3.4.2.tgz"
   integrity sha512-8drBzUEyZ2llkpCA67iYrgEssKDUu68V8ChqqOfFupIaG/LCVPUT+CoGJpT77zJprs4T/W7p07LP7zAIMuweVw==
 
+=======
+>>>>>>> main
```
Be careful, you have to remove those characters that a conflicting merge left behind, in this file its not a big problem, but be careful if this happens in an important file

BrandonVG reviewed (approved):
![image](https://github.com/BrightCoders-Institute/proyecto-final-oppenbarbies/assets/54459465/a7c41a64-2f3b-4c02-a128-75de34723924)

J3fte commented:
Buen diseño

Tole15 committed:
Resolve Duplicate Entries in CtasApp/yarn.lock

This commit addresses the duplicate entries found in the lines +3220 to +3232 of the CtasApp/yarn.lock file. The conflicting merge markers left behind were carefully removed to ensure consistency.
Changes:
- Removed conflicting merge markers in CtasApp/yarn.lock
- Ensured that the correct version of `eslint-visitor-keys` is now listed without duplication.

Tole15 committed:
Merge branch 'Sprint-2/issue-14#Create-SignIn-Screen' of https://github.com/BrightCoders-Institute/proyecto-final-oppenbarbies into Sprint-2/issue-14#Create-SignIn-Screen

Tole15 committed:
fixed HEAD comments after merge

Deleted conficts merge changes comments
